eo-maven-plugin/src/main/java/org/eolang/maven/MjLint.java

```diff
@@ -299,14 +296,10 @@ private static XML linted(
                 "empty-object",
                 "sprintf-without-formatters",
                 "inconsistent-args"
-            )
-            .defects()
-            .stream()
-            .filter(MjLint.distinctByKey(Defect::text))
-            .collect(Collectors.toList());
+            ).defects();
         defects.addAll(found);
         final Directives dirs = new Directives();
-        if (!found.isEmpty()) {
+        if (!defects.isEmpty()) {
```

Comment:
@volodya-lombrozo this is not correct. The logic here is that: we do `addIf(errors)` only if `found` defects exist, which means we should add the to xmir

Reply:
@maxonfjvipon Please, pay attention, that test are passed.